cmd/registry/cmd/upload/bulk/protos.go

```diff
@@ -395,11 +395,9 @@ func protosFromFileDescriptorSet(filename string) ([]string, error) {
 	}
 	filenameset := make(map[string]bool)
 	for _, file := range fds.File {
-		filenameset[*file.Name] = true
-		for _, dependency := range file.Dependency {
-			if !strings.HasPrefix(dependency, "google/protobuf/") {
-				filenameset[dependency] = true
-			}
+		filename = *file.Name
+		if !strings.HasPrefix(filename, "google/protobuf/") {
+			filenameset[filename] = true
```

Comment:
With `--include_imports`, we no longer need to go through the list of dependencies. All dependencies are included in the list of compiled files.